include/vctr/Containers/Span.h

```diff
@@ -102,6 +102,14 @@ class Span : public VctrBase<ElementType, std::span<ElementType, extent>, extent
         VCTR_ASSERT (extent == containerToView.size() || extent == std::dynamic_extent);
     }
 
+    /** Creates a Span that acts as a view to a compile time constant sized C style array. */
+    template <size_t n>
+    requires (n != std::dynamic_extent)
+    constexpr Span (ElementType (&cStyleArray)[n])
```

Comment:
any test for that?

Reply:
Of course! Have a look at the changes in test/TestCases/SpanConstructors.cpp